qa/rpc-tests/feature_fedpeg.py

```diff
@@ -47,27 +49,61 @@ def __init__(self):
         self.setup_clean_chain = True
         self.num_nodes = 4
 
+    def add_options(self, parser):
+        parser.add_option("--parent_binpath", dest="parent_binpath", default="",
+                          help="Use a different binary for launching nodes")
+        parser.add_option("--parent_bitcoin", dest="parent_bitcoin", default=False, action="store_true",
+                          help="Parent nodes are Bitcoin")
+
+    def start_node(self, idx):
+        is_parent = idx < 2
```

Comment:
this isn't very readable, add a comment or make it more explicit in code?

Reply:
or just get rid of the function as I propose above